app/src/main/java/com/kamron/pogoiv/Pokefly.java

```diff
@@ -1071,16 +1071,17 @@ private void populateResultsBox(IVScanResult ivScanResult) {
 
     /**
      * Adjusts expandedLevelSeekbar and expandedLevelSeekbar thumbs
-     * <p/>
+     *
      * expandedLevelSeekbar is a single thumb seekbar
      * Seekbar should be max at possible Pokemon level at trainer level 40.
      * Thumb should be placed at current Pokemon level
-     * <p/>
+     *
```

Comment:
Those are added by reformatting and make some sense if you ever actually want to look at generated JavaDocs... if you don't like them, would you mind changing that separately?